cmd/ocsp-updater/main.go

```diff
@@ -473,12 +474,13 @@ func (updater *OCSPUpdater) missingReceiptsTick(batchSize int) error {
 		if count >= updater.numLogs {
 			continue
 		}
-		cert, err := updater.sac.GetCertificate(serial)
+		cert, err := updater.sac.GetCertificate(ctx, serial)
 		if err != nil {
 			updater.log.AuditErr(fmt.Errorf("Failed to get certificate: %s", err))
 			continue
 		}
-		err = updater.pubc.SubmitToCT(cert.DER)
+		// TODO only submit to the logs we don't have a SCT for
```

Comment:
The development guidelines say that TODOs should always have an email address or issue number.